controllers/user.controller.js

```diff
@@ -78,65 +85,126 @@ const getUserById = async (req, res) => {
   } catch (error) {
     console.log(error);
     res.status(httpStatus.INTERNAL_SERVER_ERROR).json({
-      message: 'Đã sảy ra lỗi vui lòng thử lại',
+      message: 'Đã xảy ra lỗi vui lòng thử lại',
       code: httpStatus.INTERNAL_SERVER_ERROR,
     });
   }
 };
 
-const updateUserById = (req, res) => {
+const updateUserById = async (req, res) => {
   const { userId } = req.params;
-  const { fullname } = req.body;
-  const users = User.updateById(userId, { fullname });
-  if (!users) {
-    res.status(httpStatus.NOT_FOUND).json({
-      message: `Không tìm thấy người dùng`,
-      code: httpStatus.NOT_FOUND,
+  const update = req.body;
```

Comment:
```suggestion
  const updateBody = req.body;
```
đặt tên biến rõ nghĩa một chút